cypress/integration/Section/ManagedListTest.feature

```diff
@@ -0,0 +1,67 @@
+
+        Examples:
+            | url                  | indexes | title                                                                     | itemLink                                                                                                          | altText                                                                                                      | file                                                                                                                                                    |
+            | /                    | 0,1,2,3 | Overridden Title,External Card Title,Feelings and Cancer,Overridden Title | /about-cancer/coping/feelings,https://www.google.com,/about-cancer/coping/feelings,/about-cancer/coping/feelings | Sad Woman Looking Out Window,Find a Clinical Trial,Sad Woman Looking Out Window,Sad Woman Looking Out Window | sad-woman-looking-out-window-thumb.jpg,find-clinical-trial-blue-thumb.jpg,sad-woman-looking-out-window-thumb.jpg,sad-woman-looking-out-window-thumb.jpg |
+            | /about-cancer        | 0,1,2,3 | Overridden Title,External Card Title,Feelings and Cancer,Overridden Title | /about-cancer/coping/feelings,https://www.google.com,/about-cancer/coping/feelings,/about-cancer/coping/feelings | Sad Woman Looking Out Window,Find a Clinical Trial,Sad Woman Looking Out Window,Sad Woman Looking Out Window | sad-woman-looking-out-window-thumb.jpg,find-clinical-trial-blue-thumb.jpg,sad-woman-looking-out-window-thumb.jpg,sad-woman-looking-out-window-thumb.jpg |
+            | /research            | 0,1,2,3 | Overridden Title,External Card Title,Feelings and Cancer,Overridden Title | /about-cancer/coping/feelings,https://www.google.com,/about-cancer/coping/feelings,/about-cancer/coping/feelings | Sad Woman Looking Out Window,Find a Clinical Trial,Sad Woman Looking Out Window,Sad Woman Looking Out Window | sad-woman-looking-out-window-thumb.jpg,find-clinical-trial-blue-thumb.jpg,sad-woman-looking-out-window-thumb.jpg,sad-woman-looking-out-window-thumb.jpg |
+            | /about-cancer/coping | 0,1,2,3 | Overridden Title,External Card Title,Feelings and Cancer,Overridden Title | /about-cancer/coping/feelings,https://www.google.com,/about-cancer/coping/feelings,/about-cancer/coping/feelings | Sad Woman Looking Out Window,Find a Clinical Trial,Sad Woman Looking Out Window,Sad Woman Looking Out Window | sad-woman-looking-out-window-thumb.jpg,find-clinical-trial-blue-thumb.jpg,sad-woman-looking-out-window-thumb.jpg,sad-woman-looking-out-window-thumb.jpg |
+
+    Scenario Outline: Verify managed list external item's components
+        Given user is navigating to "<url>"
+        Then managed list section is visible
+        And the number of exit disclaimers is <externalLinkCount>
+        And list items at positions "<indexes>" have links as "<titleLink>"
```

Comment:
same - And list items at positions "\<indexes\>" have "links" as "<titleLink>"

Reply:
done